spec/fizz_buzz_spec.rb

```diff
@@ -1 +1,60 @@
 require "fizz_buzz"
+
+describe FizzBuzz do
+  describe ".divisible_by" do
+    context "given empty string" do
+      it "returns zero" do
+        expect(FizzBuzz.divisible_by("")).to eql(0)
+      end
+    end
+
+    context "given one" do
+      it "returns one" do
```

Comment:
Some of these contexts and its seem to say the same thing - I'd make life a bit easier and not use as many contexts!